src/assets/creds.ts

```diff
@@ -0,0 +1,2 @@
+export const client_id = '1n5qiztc3254ful7r5h1lvkxeyzqgh'
```

Comment:
Consider creating one global or several "constants" files (and not under "assets" folder)